.github/workflows/python-release.yml

```diff
@@ -7,13 +7,17 @@ on:
 jobs:
   test:
     name: Test Release
-    runs-on: self-hosted
+    
 
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
This should definitely remain true. We do not want any releases to be published for which tests are failing.